descriptors.go

```diff
@@ -66,6 +66,7 @@ func (d *descriptorSet) marshalTo(w io.Writer) error {
 
 		// TODO(stevvooe): Come up with better way to resolve this path.
 		"/usr/local/include/google/protobuf/descriptor.proto",
+		"-I/usr/local/include",
```

Comment:
This should go before the primary argument. Just switch this with the line above.